### PR TITLE
feat(app): add home screen

### LINKED ISSUE #6: Create the initial Home screen
By using the [UI/UX design](./design/prototype.gif)

- [x] Create component: **Header**
- [x] Create component: **Toolbar**

### PR DESCRIPTION
**Pull request by mehmetnyarar**
fix #6

### CODE REVIEW

File: packages/app/__tests__/app.spec [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
```
Comment (niyocodes) [inline packages/app/__tests__/app.spec:1]: This file looks redundant.

**Review by niyocodes — CHANGES_REQUESTED** [review]
Please see the comments.

File: packages/app/__tests__/app.spec (hunk 1 above)
Comment (mehmetnyarar) [reply packages/app/__tests__/app.spec]: Done.

**Review by niyocodes — APPROVED** [review]
Looks good.

**Pull request merged**